app-tasks/dags/export/do_export.py

```diff
@@ -0,0 +1,133 @@
+from __future__ import print_function
+import datetime
+import logging
+import os
+import time
+import json
+
+import subprocess
+
+from airflow.models import DAG
+from airflow.operators import BashOperator, PythonOperator
+from airflow.exceptions import AirflowException
+
+from rf.utils.io import get_session
+from rf.utils.exception_reporting import wrap_rollbar
+
+
+rf_logger = logging.getLogger('rf')
+ch = logging.StreamHandler()
+ch.setLevel(logging.INFO)
+formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+ch.setFormatter(formatter)
+rf_logger.addHandler(ch)
+
+HOST = os.getenv('RF_HOST')
+API_PATH = '/api/exports/'
+
+logger = logging.getLogger(__name__)
+
+default_args = {
+    'owner': 'raster-foundry',
+    'start_date': datetime.datetime(2017, 1, 1)
+}
+
+dag = DAG(
+    dag_id='do_export',
+    default_args=default_args,
+    schedule_interval=None,
+    concurrency=os.getenv('AIRFLOW_DAG_CONCURRENCY', 4)
+)
+
+
+
+# UTILS
+@wrap_rollbar
+def update_status(export, status):
+    url = '{0}{1}{2}/'.format(HOST, API_PATH, export)
```

Comment:
The same could be done for getting the exports at the beginning as well, which would replace another airflow task. At that point airflow would just have to execute the JAR on a schedule, but by the same token it (airflow) would lose the granularity of individual `export-id`s and associated tasks if that's something worth holding on to.